HOWTO-de.md

```diff
@@ -1,17 +1,16 @@
-Lese das hier auch in anderen Sprachen: [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md).
+Lies das hier auch in anderen Sprachen: [Français](HOWTO-fr.md), [Español](HOWTO-es.md), [简体中文](HOWTO-zh.md), [हिन्दी](HOWTO-hi.md), [繁體中文](HOWTO-zh_TW.md), [Português (BR)](HOWTO-pt_BR.md), [فارسی](HOWTO-fa_IR.md).
 
-Willkommen zu *Free-Programming-Books*! Wir heißen neue Beitragende herzlich willkommen; auch die, die ihren ersten Pullrequest (PR) auf Github vor sich haben. Wenn du einer von diesen Personen bist, dann sind hier einige nützliche Ressourcen:
+Willkommen zu *Free-Programming-Books*! Wir heißen neue Beitragende herzlich willkommen. Auch die, die ihren ersten Pull Request (PR) auf GitHub machen möchten. Wenn Du eine dieser Personen bist, dann findest Du hier einige nützliche Ressourcen:
 
 * [Informationen zu Pull Requests](https://docs.github.com/de/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
-* [Pull Request erstellen](https://docs.github.com/de/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
-* [Github Hallo Welt](https://guides.github.com/activities/hello-world/) *(auf Englisch)*
-* [Youtube - Tutorial Github für Anfänger](https://www.youtube.com/watch?v=0fKg7e37bQE) *(auf Englisch)*
-* [Youtube - So forkst du ein Github-Repo und sendest einen Pull-Request](https://www.youtube.com/watch?v=G1I3HF4YWEw) *(auf Englisch)*
-* [Youtube - Github Markdown](https://www.youtube.com/watch?v=HUBNt18RFbo) *(auf Englisch)*
+* [Pull Requests erstellen](https://docs.github.com/de/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
```

Comment:
"Pull Request erstellen" is the original title of the article at https://docs.github.com/de/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request - so I think this should stay?

Reply:
Yup, I was being misled by the "...work-with-pull-requests" in the URL